resources/js/components/catalog-new/NumberFormatter.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    {{
+        value
+            .toLocaleString('en-US', {
+                useGrouping: true,
+            })
+            .replace(/,/g, ' ')
+    }}
```

Comment:
This is a bit awkward to read, could you please move it into `computed`?

I'd also change the locale so that we don't have to remove the comma:

```suggestion
    {{
        value
            .toLocaleString('sk-SK', {
                useGrouping: true,
            })
    }}
```